tsconfig.json

```diff
@@ -17,9 +17,8 @@
         "outDir": "./build"
     },
     "files": [
-        "export-to-csv.ts",
-        "export-to-csv.spec.ts",
-        "index.ts"
+        "src/export-to-csv.ts",
+        "src/index.ts"
```

Comment:
Not all files are included. I believe that this "files" section is not needed, 'exclude' should be enough.